scripts/windows/windows-prepare.ps1

```diff
@@ -69,3 +69,13 @@ Write-Output "Enabling Remote Desktop..."
 Set-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Control\Terminal Server" -Name "fDenyTSConnections" -Value 0 | Out-Null
 Set-ItemProperty -Path 'HKLM:\System\CurrentControlSet\Control\Terminal Server\WinStations\RDP-Tcp' -name "UserAuthentication" -Value 0
 Enable-NetFirewallRule -Group '@FirewallAPI.dll,-28752'
+
+# Check if the OS is Windows 11
+if ((Get-ComputerInfo | Select-Object -expand OsName) -match 11) {
+    # Remove Appx Packages causing Sysprep to fail - https://kb.vmware.com/s/article/82532
+    Write-Output "Removing Appx Packages causing Sysprep to fail..."
+    Write-Output "Setting the Non-Removable App Policy for TikTok..."
```

Comment:
This is limited to `1.0.5.0_neutral__6yccndn6064se" - what if the media is updated and a newer version is added - for example, a "Windows 11 H123". 

Is there a more efficient method rather than pinning the version?

Reply:
More than happy to give it a whirl with the Jan update of Windows 11 and see if it reproduces. Let me test and see if it fails on the same place

Reply:
Thanks, appreciate that.

Reply:
Hi @LukeEvansTech - any update?

Reply:
@tenthirtyam, I'm testing the Jan update this week. Should have feedback in the coming days.

Reply:
❓ Hi @LukeEvansTech - any update?

Reply:
> @tenthirtyam, I'm testing the Jan update this week. Should have feedback in the coming days.

❓ Hi @LukeEvansTech - any update?

Reply:
Hey @tenthirtyam, I tested this on en-us_windows_11_business_editions_version_22h2_updated_feb_2023_x64_dvd_4e45f23d.iso - it built successfully and even better it deployed successfully without the need for the above!

Should be ok to close this one down with the PR.

Reply:
If you’d like to put in a PR to bump Win11 to February 23 please feel free to do so!